spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :system do
+  let!(:user1) do
+    User.create(
+      name: 'test user1',
+      photo: 'https://images3.alphacoders.com/690/690494.jpg',
+      bio: 'test_bio1',
+      posts_counter: 1
+    )
+  end
+
+  let!(:posts) do
+    [
+      Post.create(author: user1, title: 'Post 1', text: 'Post 1 content', comments_counter: 1, likes_counter: 1),
+      Post.create(author: user1, title: 'Post 2', text: 'Post 2 content', comments_counter: 2, likes_counter: 2),
+      Post.create(author: user1, title: 'Post 3', text: 'Post 3 content', comments_counter: 3, likes_counter: 3)
+    ]
+  end
+
+  describe 'show user and post details' do
+    before(:example) do
+      visit user_path(user1)
+    end
+
+    it 'displays user information' do
+      expect(page).to have_content(user1.name)
+      expect(page).to have_content(user1.posts_counter)
+      expect(page).to have_content(user1.bio)
+      expect(page).to have_selector("img[src='#{user1.photo}']")
```

Comment:
- [ ] Please assert the test for the profile picture of the user differently not in the single `it` block. Because, in case of test failure it would be difficult to know exactly what happened and why. So that's why please every assetion should in their respective `it` block.